src/assets/keycloak/keycloak.md

```diff
@@ -0,0 +1,68 @@
+---
+title: keycloak
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: March 26, 2024
+tags: Keycloak
+banner: https://blog.jankaritech.com/src/assets/keycloak/images/keycloak.png
+---
+
+## Keycloak
+Keycloak is an [open-source](https://github.com/keycloak/keycloak) project for Red Hat SSO. It provides `Identity and Access Management` solution designed to secure application services.
```

Comment:
```suggestion
Keycloak is an [open-source](https://github.com/keycloak/keycloak) project for Red Hat SSO. It provides an `Identity and Access Management` solution designed to secure application services.
```

or 
```suggestion
Keycloak is an [open-source](https://github.com/keycloak/keycloak) project for Red Hat SSO. It provides `Identity and Access Management` solutions designed to secure application services.
```
if this is singular, I think this needs an article. Check with the English teacher or with @phil-davis :)